### PR TITLE
my first ever contribution

### PR DESCRIPTION
**Pull request by nausherofficial**
thank you anuj garg and code for cause team to provide me this opportunity to contribute a open source project

### CODE REVIEW

File: src/data/contributors/nausherali.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Nausher Ali",
+    "skills": ["MERN", "Django", "Spring"],
+    "cause": "serve humanity ",
```
Comment (Abhishek-kumar09) [inline src/data/contributors/nausherali.json:4]: What a great cause to code :) Just a small suggestion.
image
I think `I code for serving humanity` is gramatically more accurate.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

**Comment by nausherofficial** [comment]
Got it

**Review by nausherofficial — COMMENTED** [review]
changes applied

**Review by Abhishek-kumar09 — APPROVED** [review]

**Comment by Abhishek-kumar09** [comment]
Thanks for your PR @nausherofficial . Welcome to @codeforcauseorg.

**Comment by nausherofficial** [comment]
> Thanks for your PR @nausherofficial . Welcome to @codeforcauseorg.

thank you so much @Abhishek-kumar09 .

**Pull request merged**